front/src/styles/scss/applications/operationalStudies/_simulationresults.scss

```diff
@@ -48,6 +48,17 @@
     }
   }
 
+  .simulation-sheet-container {
+    padding-inline: 42.5px 33.5px;
+    display: flex;
+    justify-content: space-between;
+    padding-top: 1rem;
+    button {
+      font-weight: 500;
+      background-color: rgba(235, 235, 234, 0.4);
```

Comment:
For consistency, we should add a var for this color in hex. We could name it `grey10O40: #ebebea66` (O for opacity)

Reply:
I just add a comment between bg color